.travis.yml

```diff
@@ -42,6 +42,7 @@ before_install:
 install:
   - pip install --upgrade pip setuptools
   - pip install -r requirements/base.pip
+  - pip install -r requirements/s3.pip
```

Comment:
Does this mean the base installation will require the s3 packages?

Reply:
Yes I believe so, the travis tests kept failing because of s3 storages